product_docs/docs/bart/2.5/bart_inst/index.mdx

```diff
@@ -22,7 +22,8 @@ This guide provides information about how to install and configure the EDB Backu
 -   To view a complete list of platforms that EDB supports, visit the [EDB website.](https://www.enterprisedb.com/services-support/edb-supported-products-and-platforms)
 
 !!! Note
-    BART 2.5 does not support CentOS/RHEL/OEL 6.x platforms. EDB recommends migrating the EDB products running on these platforms to a supported platform.
+
+BART 2.5 is no longer supported on CentOS/RHEL/OL 6.x platforms. It is strongly recommended that EDB products running on these platforms be migrated to a supported platform.
```

Comment:
Please check the alignment of the updated line.

Reply:
Thanks for pointing this out Abhilasha. I have aligned the note